### ISSUE TITLE
Test human-entered div tags

### ISSUE THREAD

**Issue by zkamvar** [carpentries/pegboard#10]
So far, in #9, we are dealing with our generated div tags, but user-entered div tags are not always so clean or predictable. 

Case in point: commonmark will wrap all consecutive HTML tags in its own block, which means that you can have something like this:

```html
<div class="row">
  <div class="col-md-6" markdown="1">
   <!--... content ...-->
  </div>
  <div class="col-md-6" markdown="1">
   <!--... content ...-->
</div>
```

Commonmark parses this a **three elements**,  lines 1 and 2, lines 4 and 5, and line 7

The way #9 is parsing divs assumes that opening and closing tags are independent. We could fix this by splitting up the mutli-tags and placing them in their own `html_block` elements.

**Comment by zkamvar** [comment]
This was addressed in 9bbad49

**Comment by zkamvar** [comment]
This was tested here: https://zkamvar.github.io/glowing-chainsaw/08-survey-results.html
